storage/store.go

```diff
@@ -472,7 +472,7 @@ func (s *Store) Start() error {
 		s.gossip.RegisterCallback(gossip.KeyConfigAccounting, s.configGossipUpdate)
 		s.gossip.RegisterCallback(gossip.KeyConfigZone, s.configGossipUpdate)
 		// Callback triggers on capacity gossip from all stores.
-		capacityRegex := fmt.Sprintf("%s.*", gossip.KeyMaxAvailCapacityPrefix)
+		capacityRegex := gossip.MakePrefixPattern(gossip.KeyMaxAvailCapacityPrefix)
```

Comment:
Not sure if this is any clearer. I didn't mind the `fmt.Sprintf` or you could just concatenate the strings:

```
  capacityRegex := gossip.KeyMaxAvailCapacityPrefix + ".*"
```